check_pgactivity

```diff
@@ -5551,6 +5558,18 @@ sub check_streaming_delta {
 
     @rs = @{ query_ver( $hosts[0], %queries ) };
 
+    my $index = 0;
```

Comment:
I think it'd be better to handle the excluded connections in the "foreach my $host (@rs) {" loop.

I guess you did it this way to get the number of non excluded connections, but I think the "# of slaves" perfdata should return the real number of connections.